code/game/machinery/turrets.dm

```diff
@@ -6,7 +6,7 @@
 	var/enabled = 1
 	var/obj/item/weapon/gun/installed = null	// the type of weapon installed
 	anchored = 1
-	invisibility = INVISIBILITY_LEVEL_TWO		// the turret is invisible if it's inside its cover
+	invisibility = INVISIBILITY_LEVEL_ONE		// the turret is invisible if it's inside its cover
```

Comment:
Isn't this intended as the turret machine being very invisible while inside the cover as the comment states? Not really sure reducing it to invisible one makes sense.

Reply:
Turrets are important to AIs, and I'm not sure how relevant it is to them to see them, but I changed it so that they can still see them. They're still invisible to most things.